debug_toolbar/static/debug_toolbar/css/toolbar.css

```diff
@@ -615,16 +615,35 @@
     margin-bottom: 3px;
     font-family:Consolas, Monaco, "Bitstream Vera Sans Mono", "Lucida Console", monospace;
 }
+#djDebug .djdt-stack {
+	white-space: normal;
+}
 #djDebug .djdt-stack span {
-    color: #000;
```

Comment:
this file is inconsistent with indents. some are tabs and some are spaces. let me know if I should make it consistent in whole file.